src/Microsoft.CodeQuality.Analyzers/Core/ApiDesignGuidelines/ImplementIDisposableCorrectly.cs

```diff
@@ -381,10 +381,12 @@ private void CheckDisposeImplementationRule(IMethodSymbol method, INamedTypeSymb
                 }
             }
 
-            /// <summary>
-            /// Checks rule: Modify {0} so that it calls Dispose(false) and then returns.
-            /// </summary>
+#pragma warning disable CA1801 // Review unused parameters
```

Comment:
Did the suppression code fix change the formatting of doc comments here?

Reply:
Must have.

---
In reply to: [150944282](https://github.com/dotnet/roslyn-analyzers/pull/1356#discussion_r150944282) [](ancestors = 150944282)